arma-rs/src/context/mod.rs

```diff
@@ -89,8 +87,11 @@ impl Context {
     where
         V: IntoArma,
     {
-        self.queue
-            .push((name.to_string(), func.to_string(), Some(data.to_arma())));
+        let _ = self.callback_tx.send(CallbackMessage::Call(
+            name.to_string(),
+            func.to_string(),
+            Some(data.to_arma()),
+        ));
```

Comment:
I do not like these just dropping the error, they should pass it up to the caller

Reply:
Can do, could indeed be handy for testing. Note that when the extension is used with Arma calling `send` at shutdown still results in a *STATUS_ACCESS_VIOLATION*, I don't think there's anyway to prevent this.

Additionally while we're at it, it might be a good opportunity to remove the deprecated callback method?

Reply:
Addressed in [a1bd5a7](https://github.com/BrettMayson/arma-rs/pull/40/commits/a1bd5a7530aee40d1fcf691d60b655c517ea523c), and I also took the opportunity to remove `callback` from the public methods. With the introduction of error handling, I think the changes were significant enough that the old `callback` doesn't really fit anymore.